third-party/chpl-venv/Makefile

```diff
@@ -7,12 +7,12 @@ include $(CHPL_MAKE_HOME)/make/Makefile.base
 include $(THIRD_PARTY_DIR)/chpl-venv/Makefile.include
 
 ifneq (,$(findstring cygwin, $(CHPL_MAKE_TARGET_PLATFORM)))
-# easy_install from cygwin's setuptools is `easy_install-<MAJOR>.<MINOR>`
-# instead of `easy_install`. If we can't find easy_install, look for the
+# pip from cygwin's setuptools is `pip-<MAJOR>.<MINOR>`
+# instead of `pip`. If we can't find pip, look for the
 # versioned one.
-EASY_INSTALL = $(shell which easy_install 2>/dev/null || which easy_install-$(PYTHON_VERSION))
+PIP = $(shell which pip 2>/dev/null || which pip-$(PYTHON_VERSION))
```

Comment:
Looks like it's `pip<version>` instead of `pip-<version>`, so this should be:

```sh
PIP = $(shell which pip 2>/dev/null || which pip$(PYTHON_VERSION))
```